src/Symfony/Component/Console/Application.php

```diff
@@ -615,10 +638,10 @@ public static function getAbbreviations($names)
     /**
      * Renders a caught exception.
      *
-     * @param \Exception      $e      An exception instance
-     * @param OutputInterface $output An OutputInterface instance
+     * @param \Throwable|\Exception $e      An throwable instance
+     * @param OutputInterface       $output An OutputInterface instance
      */
-    public function renderException(\Exception $e, OutputInterface $output)
+    public function renderException(/*\Throwable */$e, OutputInterface $output)
```

Comment:
removing the typehint is a BC break for classes extending the method (and there are such classes out there)